client/trunk.lua

```diff
@@ -248,29 +248,27 @@ CreateThread(function()
                         else
                             QBCore.Functions.Notify(Lang:t("error.trunk_closed"), 'error', 2500)
                         end
-                        sleep = 100
+                        Wait(100)
                     end
                     if GetVehicleDoorAngleRatio(vehicle, 5) > 0 then
-                        sleep = 0
```

Comment:
Why remove this?

Reply:
We only need to sleep 0 at the first 3DsText, since we have the Wait in the key presses, it won't affect any other 3DsText.